yml/apdb.yaml

```diff
@@ -2177,88 +2178,88 @@ tables:
     datatype: float
     description: Estimated sky background at the position (centroid) of the object.
     mysql:datatype: FLOAT
-    fits:tunit: nJy/asec^2
+    fits:tunit: nJy/arcsec^2
```

Comment:
The same error shows up in `yml/imsim.yaml`, `yml/dp01_dc2.yaml`, `yml/dp02_dc2.yaml`. Even if you can't test them, why not do the search-and-replace?

Reply:
I've fixed `imsim.yaml`, but I don't want to modify the dp01/dp02 files, because they're associated with already-released data products and I believe are under a more strict modification process. I'll leave it to @gpdf to sort out those two on the other tickets about this.